cores/esp32/Tone.cpp

```diff
@@ -6,20 +6,18 @@
 
 static TaskHandle_t _tone_task = NULL;
 static QueueHandle_t _tone_queue = NULL;
```

Comment:
Not sure, but I think that @me-no-dev want us to try to do not use Tasks within the peripherals.
I don't see a clear motivation for using a task in the `Tone` implementation...

Reply:
Maybe @PilnyTomas can take a look on this in separate PR, as he wrote this implementation :)

Reply:
Motivation here is to be able to "setup" a melody and let it play, without having to wait for it to be over. Same goes for when you want to just set it to beep for 100ms and not having to wait. We could look into doing it blocking or maybe coming up with another idea.